docs/general/ledger.md

```diff
@@ -15,11 +15,15 @@ browsers like Brave and Chrome.
 :::
 
 The Polkadot [Ledger][] application is compatible with both the Ledger Nano S and the Ledger Nano X
-devices. Ledger devices are hardware wallets that keep your secret secured on a physical device that
-does not expose it to your computer or the internet.
+devices. Ledger devices are hardware wallets that keep your secret key secured on a physical device that
+does not expose it to your computer or the internet. That is, even if you connect your nano via USB to your computer, the private keys will not be leacked. Ledger devices are also hierarchical deterministic wallets (HD wallets), that is:
 
+  - Deterministic means that there is only one seed phrase to generate all the accounts for different cryptocurrencies.
+  - Hierarchical means that the accounts are generated in a tree-like structure for different purposes.
+
+Ledger devices can be equipped with applications that are blockchain-specific. Such applications are usually develped by third parties and they enable the user to perform essential actions such as transacting and staking.
 The Polkadot Ledger application allows you to manage Polkadot's native token, DOT. It supports
-most of the transaction types of the network ,including batch transactions from the Utility pallet.
+most of the transaction types of the network, including batch transactions from the Utility pallet. Note that the Polkadot Ledger application does not currently support the staking extrinsic `rebag`.
```

Comment:
Adding more context and linking to a resource where the reader can find the supported transactions on Polkadot Network. Also, feel free to use https://github.com/Zondax/ledger-polkadot whenever you like to highlight something to the reader.

```suggestion
most of the transaction types of the network, including batch transactions from the Utility pallet. 

:::note Ledger apps may not support all the transactions

Check [Ledger Polkadot App](https://github.com/Zondax/ledger-polkadot) specification for the list of transactions supported. It could be possible that some transactions are supported only on a specific version of the app and some transactions are not supported at all by any version. For instance, the staking `rebag' extrinsic is supported on the XL version but not on the light version. The `setIdentity' extrinsic is not supported by any of the app versions.

:::

```